runtime-library/stdlib/lib_array.c

```diff
@@ -1738,6 +1770,8 @@ static NativeSymbol native_symbols[] = {
     REG_NATIVE_FUNC(array_includes_i64, "(rrIr)i"),
     REG_NATIVE_FUNC(array_includes_i32, "(rrir)i"),
     REG_NATIVE_FUNC(array_includes_anyref, "(rrrr)i"),
+
+    REG_NATIVE_FUNC(arraybuffer_slice, "(rrFr)r"),
```

Comment:
Is it possible to reuse array slice?

Reply:
I supposed no. Their signature's parameter type is not equal.

Arraybuffer's slice signature is `slice(begin: number, end?: number): ArrayBuffer;`, which is `(rrFr)r`. (After we implement wasmType, it will be `(rrir)r`).
array's slice signature is `slice(start?: number, end?: number): T[];`, which is `(rrrr)r`.

Reply:
The `begin` parameter should also be optional parameter, where did you see that it's mandatory?

Reply:
`lib.es5.d.ts` shows that `begin` is mandatory:
https://github.com/microsoft/TypeScript/blob/eb374c28d6810e317b0c353d9b1330b0595458f4/src/lib/es5.d.ts#L1663-L1673